src/adapters/ActivitiesJSONAdapter.js

```diff
@@ -0,0 +1,45 @@
+import {Observable} from 'rxjs';
+
+import ActivitiesAdapter from './ActivitiesAdapter';
+
+export default class ActivitiesJSONAdapter extends ActivitiesAdapter {
```

Comment:
We should add a docstring to this class. Something like:
```
/*
 * Implements the ActivitiesAdapter interface with a JSON object as its datasource. See @ActivitiesJSON
 */
```

We should probably do the same on `PeopleJSONAdapter`.